src/models/sqsPayload.ts

```diff
@@ -5,11 +5,13 @@ export interface SQSRequestBody {
   techRecord: object;
   plate?: Plates;
   letter?: Letter;
+  adrCertificate?: any;
```

Comment:
The typescript inference in the one place you're using this would suggest you want this:
```suggestion
  adrCertificate?: ADRCertificateDetails;
```

Reply:
thanks i missed this, was from when I was doing things quick